code/game/gamemodes/devil/devil_game_mode.dm

```diff
@@ -75,3 +76,22 @@
 	var/datum/antagonist/devil_datum = L.mind.has_antag_datum(ANTAG_DATUM_DEVIL)
 	devil_datum.on_removal()
 	return TRUE
+
+/datum/game_mode/devil/generate_credit_text()
+	var/list/round_credits = list()
+	var/len_before_addition
+
+	round_credits += "<center><h1>The Tempting Devils:</h1></center>"
+	len_before_addition = round_credits.len
+	var/datum/antagonist/devil/devil_info
+	for(var/datum/mind/devil in devils)
+		devil_info = devil.has_antag_datum(ANTAG_DATUM_DEVIL)
+		if(devil_info) // This should never fail, but better to be sure
+			round_credits += "<center><h2>[devil_info.truename] in the form of [devil.name]</h2>"
+			devil_info = null
+	if(len_before_addition == round_credits.len)
+		round_credits += list("<center><h2>The devils were all utterly destroyed!</h2>", "<center><h2>The love of God shines through!</h2>")
```

Comment:
Not comfortable with the "God" bit. While I think the Devils are garbage in terms of lore, adding stuff about God to here will only make it weirder.